app/views/posts/_expanded.html.erb

```diff
@@ -448,8 +448,7 @@
                 <% elsif post.comments_disabled %>
                   <p class="has-color-red-500"><i class="fa fa-lock"></i> Comments have been disabled on this post, but as a moderator you are exempt from that block.</p>
                 <% end %>
-<!--                <a href="#" class="button is-muted is-small" data-modal=".js--<%#= post.id %>-new-thread-modal">Start new comment thread</a>-->
-                <a class="button is-muted is-small" onclick="new_thread(<%= post.id %>)">Start new comment thread</a>
+                <a class="button is-muted is-small js-new-thread-link" data-post="<%= post.id %>">Start new comment thread</a>
```

Comment:
You've changed this to a class and removed the `onclick` attribute, which is great, but I don't see anywhere you've linked it up in the JS? Refer to my original review comment for an example - you need a click handler in the JS.

Reply:
I guess, you had changed it earlier. Cause, your suggested text is showing to me. Maybe, you have requested the changes from first/second one. 
![Screenshot from 2021-07-25 00-36-34](https://user-images.githubusercontent.com/58106197/126887906-3bfe3cdf-8c38-462c-9a91-ffe8a534e762.png)

(https://github.com/Istiakshovon/qpixel/edit/develop/app/views/posts/_expanded.html.erb?pr=%2Fcodidact%2Fqpixel%2Fpull%2F563)

Reply:
I have edited it's working now.. (tested)